src/DotLiquid.Tests/FilterTests.cs

```diff
@@ -360,18 +356,16 @@ public void TestFilterInContextOverridesMethodWithSameMethodSignaturesDifferentC
             Assert.AreNotEqual("ABClass One", new Variable("'A' | concatenate : 'B'").Render(_context));
         }
 
-
         [Test]
         public void TestFilterAsLocalOverridesMethodWithSameMethodSignaturesDifferentClasses()
         {
             Helper.AssertTemplateResult(
-                           expected: "ABClass Two",
+                           expected: "ABClass One",
                            template: "{{'A' | concatenate : 'B'}}",
                            localVariables: null,
-                           localFilters: new[] { typeof(FilterWithSameMethodSignatureDifferentClassOne), typeof(FilterWithSameMethodSignatureDifferentClassTwo) });
+                           localFilters: new[] {  typeof(FilterWithSameMethodSignatureDifferentClassTwo), typeof(FilterWithSameMethodSignatureDifferentClassOne) });

```

Comment:
nit: @daviburg won't like this change to two spaces.

Reply:
Fixed: https://github.com/dotliquid/dotliquid/pull/453/commits/10a524c01c5287c9ddfd6a794e12a8371917e549 
I'm sure @daviburg will appreciate it 😸